recipes/cosmosis-standard-library/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "cosmosis-standard-library" %}
+{% set version = "2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/joezuntz/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: fc815e5068a1e212c1af8af2106c42e14881f466b462c0a6c918a144e5fbb30c
+
+build:
+  number: 0
+  skip: True  # [win or (py < 36)]
+  detect_binary_files_with_prefix: True
+  missing_dso_whitelist:
+    - "*/liblimber.so"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('cxx') }}
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - python
+    - cosmosis
+    - cython
+    - mpich          # [mpi == "mpich"]
+    - openmpi        # [mpi == "openmpi"]
+    - libblas
+    - libcblas
+    - liblapack
+    - cfitsio
+    - gsl
+    - fftw
+    - scipy
+    - astropy
+    - numpy
+    - matplotlib
+    - fitsio
+    - pyyaml
+    - scikit-learn
+    - fast-pt
+  run:
+    - python
+    - {{ pin_compatible('cosmosis', max_pin='x.x') }}
+
+test:
+  commands:    
+    - test -f $PREFIX/lib/cosmosis-standard-library/likelihood/planck2018/planck_interface.so
+
+about:
+  home: https://github.com/joezuntz/cosmosis-standard-library
+  summary: 'CosmoSIS is a cosmological parameter estimation code.'
+  description: |
+    It is a framework for structuring cosmological parameter estimation with a
+    focus on flexibility, re-usability, debugging, verifiability, and code 
+    sharing in the form of calculation modules. It consolidates and connects 
+    together existing code for predicting cosmic observables, and makes mapping
+    out experimental likelihoods with a range of different techniques much more
+    accessible.
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
```

Comment:
The license file here is incorrect. We cannot simply apply random licenses to other people's code and then distribute it on conda-forge. As far as I can tell, the upstream package has no license. @joezuntz will need to figure out the correct license for the upstream code and apply the license himself there before we can merge, this.

Reply:
This is very complicated, and is one reason I have avoided trying to share a compiled version of the CSL.

The different bits of included code are under different licenses. Some are standard licenses, but there are many different ones, GPL, LGPL, BSD, MIT. 

Some are vague, like  "You can use X freely, provided that in your publications, you cite at least the paper ..." or similar generic statements.

I understand it is permitted or acceptable to share the source code of all of them, but I would not recommend trying to sort through this to the level needed here.

There is already a pacakge `cosmosis-build-standard-library` which downloads and builds the CSL for the user. I would strongly recommend using that instead, especially as most users of CSL are also developers.  It supplies a one-line command to download and build the CSL.  If more is needed then we could add something to that command to install the result to a central location.